Tests/scripts/collect_tests/utils.py

```diff
@@ -171,8 +171,13 @@ def __init__(self, path: Path):
     @property
     def _has_no_id(self):
         # some content files may not have an id
-        return self.path.name == 'pack_metadata.json' or self.path.name.endswith(
-            '_schema.json') or self.path.name.endswith('testdata.json')
+        file_path_splitted = self.path.parts
+        return self.path.name == 'pack_metadata.json' \
+            or self.path.name.endswith('_schema.json') \
+            or self.path.name.endswith('testdata.json') \
+            or len(file_path_splitted) > 1 \
+                and file_path_splitted[-2].lower().startswith('releasenotes') \
+                and file_path_splitted[-1].lower().endswith('.json')
```

Comment:
what cases are caught here?

Reply:
release notes config files are json files in the ReleaseNotes folder.

Reply:
then the best way is `ReleaseNotes in path.parts and path.suffix == '.json'`